docs/content/installation/uninstall_driver_openshift_web.md

```diff
@@ -11,12 +11,18 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 
     **Operators** > **Installed Operators** > **Operator Details**.
 
-4.  Click on the **more** menu for the **ibm-block-csi** driver and select **Delete IBMBlock CSI**.
+4. If applicable, click on the **more** menu for the **ibm-block-csi_hostdefiner** plugin and select **Delete IBMBlock CSI HostDefiner**.
+
+    Wait for the controller and node pods to terminate.
```

Comment:
```suggestion
    Wait for the host definer pod to terminate.
```

Reply:
updated

note - i'm still waiting on the actual name of the host definer  plugin and Delete button on openshift web here from @matancarmeli7